crates/bevy_ui/src/render/mod.rs

```diff
@@ -145,13 +145,47 @@ fn get_ui_graph(render_app: &mut App) -> RenderGraph {
     ui_graph
 }
 
+/// Texture size and subrect data
+#[derive(Clone, Copy, Debug)]
+pub enum ExtractedRect {
```

Comment:
This makes the extracted node bigger due to the addition of the enum discriminator. This could have a negative impact on extraction perf.

Reply:
They both are both the same size, aren't they? The optional atlas_size field is removed.